README.md

```diff
@@ -266,6 +268,39 @@ Example:
 kubergrunt eks sync-core-components --eks-cluster-arn EKS_CLUSTER_ARN
 ```
 
+#### cleanup-security-group
+This subcommand cleans up the leftover AWS-managed security groups that are associated with an EKS cluster you intend
+to destroy. It accepts
+- `--eks-cluster-arn`: the ARN of the EKS cluster
+- `--security-group-id`: the AWS-managed security group ID
+- `--vpc-id`: the VPC ID where the cluster is located
+
+It also looks for other security groups associated with the EKS cluster, such as the AWS-managed security group that
+is created for a load balancer controller. To safely delete these resources, it detaches and deletes any associated
+elastic network interfaces.
+
+Example:
+
+```bash
+kubergrunt eks cleanup-security-group --eks-cluster-arn EKS_CLUSTER_ARN --security-group-id SECURITY_GROUP_ID \
+--vpc-id VPC_ID
+```
+
+#### schedule-coredns
+This subcommand allows for Fargate and EC2 changes when you create or destroy an EKS cluster. During the creation of
+an EKS cluster that uses Fargate, `schedule-coredns fargate` will annotate the deployment so that CoreDNS can find
+and allow EKS to use Fargate nodes. When destroying a Fargate-enabled EKS cluster, `schedule-coredns ec2` can be run
+to reset the annotations such that EC2 nodes can be found by CoreDNS. Currently `fargate` and `ec2` are the only sub-
+commands that `schedule-coredns` accepts.
```

Comment:
NIT: I would suggest decoupling the use case for the command, from what the command does. E.g., 

```suggestion
and allow EKS to use Fargate nodes. To switch back to EC2, you can run `schedule-coredns ec2` to reset the annotations such that EC2 nodes can be found by CoreDNS.

This command is useful when creating Fargate only EKS clusters. By default, EKS will schedule the `coredns` service assuming EC2 workers. This command can be used to force the service to run on Fargate.

Currently `fargate` and `ec2` are the only sub-commands that `schedule-coredns` accepts.

This command is also designed to be used as `local-exec` provisioners on an `aws_eks_fargate_profile` resource so you can schedule the CoreDNS service after creating the profile, and revert back when destroying the profile.
```